### ISSUE TITLE
Adding document and Wiki translations

### ISSUE THREAD

**Issue by yolowex** [libgdx/libgdx.github.io#149]
I can translate the libGDX wiki and documentation into Persian (فارسی).

I believe this would be a valuable contribution to both the libGDX community and Persian developers who may not have a strong command of English.

I have discussed this idea with some contributors on the Discord server, and Damios suggested that I open an issue here.

**Comment by crykn** [comment]
Hey @yolowex!

It's very kind of you to offer to translate the wiki! As I've already written on the Discord server, there are two things we'd need to figure out first:

- Is there a good technical solution for doing this with Jekyll? A quick look didn't turn up any good out-of-the-box solutions, but I think this shouldn't  be too difficult if we adapt the wiki template file. 
- Do we even want multi-language support for the wiki? That would certainly increase the work needed for maintaining the site and could sooner or later lead to one of the versions of the wiki being outdated. This all depends on finding people willing to translate new changes in a timely manner. 

Since this is quite a big decision, I'd certainly appreciate the input of other people on this!

**Comment by yolowex** [comment]
For the second question, I believe yes.
It's obvious that anyone who wants to learn programming and advance in it, has to know English.

But for many people who are just starting tolearn programing, aren't as fluent as needed  to read the docs.

Ofc they have to learn English eventually, but in the early stages of their learning course (programming), this can be a such a barrier that has the potential to easily reduce their motivation.

So in short, I think having the docs translated makes the framework more beginner friendly, and is beneficial for both the newbies and the framework (in the long run).

**Comment by Frosty-J** [comment]
Even in English, the wiki is lacking maintainers. Some of the Android stuff in particular is very old, circa 2009 I reckon, which is an eternity when Android's best practices change annually. I'm not against manual translations, but it would be a challenge to maintain (there's even more wiki than it appears at first glance - see #70).

**Comment by obigu** [comment]
I'd agree better focus the efforts on better mantaining the English wiki than adding  a new translation which would quickly become out of sync with the English one. 

@yolowex Imo a way to spread libGDX in your community/language would be to create your own content on that language and we could link it from the libGDX site if we consider it useful.

**Comment by crykn** [comment]
Considering the maintenance required for this, the consensus seems to be that non-english resources are better suited for third-party sites. If there ever are any serious translation projects, we can link to them on the starting page of the wiki.
